scripts/content.js

```diff
@@ -45,56 +62,63 @@ function callTranslateAPI(text) {
  * @param  {object} mutationsList  text element object.
  * @param  {object} observer  observer properties.
  */
-function translateComments(mutationsList, observer) {
+async function translateComments(mutationsList, observer) {
   for (const mutation of mutationsList) {
     if (mutation.type == "childList") {
-      mutation.addedNodes.forEach((node) => {
+      for (const node of mutation.addedNodes) {
         if (node.nodeName.toLowerCase() == "ytd-comment-renderer") {
```

Comment:
Why is the `toLowerCase` method required here?

Reply:
good question, will remove

Reply:
`toLowerCase` helps with readibility, the classes are written in lowercase, so just helps with matching

alternatively, we can just do: `node.nodeName == "YTD-COMMENT-RENDERER"` but that is ugly, let me know which one you want

Reply:
yeah u right caps is ugly.